web/src/scheduled_messages.ts

```diff
@@ -33,38 +46,48 @@ function compute_send_times(now = new Date()) {
     return send_times;
 }
 
-export function add_scheduled_messages(scheduled_messages) {
+export function add_scheduled_messages(scheduled_messages: ScheduledMessage[]): void {
     for (const scheduled_message of scheduled_messages) {
         scheduled_messages_data.set(scheduled_message.scheduled_message_id, scheduled_message);
     }
 }
 
-export function remove_scheduled_message(scheduled_message_id) {
+export function remove_scheduled_message(scheduled_message_id: number): void {
     if (scheduled_messages_data.has(scheduled_message_id)) {
         scheduled_messages_data.delete(scheduled_message_id);
     }
 }
 
-export function update_scheduled_message(scheduled_message) {
+export function update_scheduled_message(scheduled_message: ScheduledMessage): void {
     if (!scheduled_messages_data.has(scheduled_message.scheduled_message_id)) {
         return;
     }
 
     scheduled_messages_data.set(scheduled_message.scheduled_message_id, scheduled_message);
 }
 
-export function delete_scheduled_message(scheduled_msg_id, success = () => {}) {
-    channel.del({
+export function delete_scheduled_message(
+    scheduled_msg_id: number,
+    success = () => {
+        /* do nothing*/
```

Comment:
[eslint](https://eslint.org/docs/latest/rules/no-empty-function). Can this be handled differently?